contrib/dyn_templates/src/lib.rs

```diff
@@ -435,3 +452,104 @@ impl Sentinel for Template {
         false
     }
 }
+
+/// A macro to easily create a template rendering context.
+///
+/// Invocations of this macro expand to a value of an anonymous type which
+/// implements [`serde::Serialize`]. Fields can be literal expressions or
+/// variables captured from a surrounding scope, as long as all fields implement
+/// `Serialize`.
+///
+/// # Examples
+///
+/// The following code:
+///
+/// ```rust
+/// # #[macro_use] extern crate rocket;
+/// # use rocket_dyn_templates::{Template, context};
+/// #[get("/<foo>")]
+/// fn render_index(foo: u64) -> Template {
+///     Template::render("index", context! {
+///         // Note that shorthand field syntax is allowed.
+///         // This is the same as `foo: foo,`
+///         foo,
+///         bar: "Hello world",
+///     })
+/// }
+/// ```
+///
+/// is equivalent to the following, but without the need to manually define an
+/// `IndexContext` struct:
+///
+/// ```rust
+/// # #[macro_use] extern crate rocket;
+/// # use rocket_dyn_templates::Template;
+/// # use serde::Serialize;
```

Comment:
```suggestion
/// # use rocket::serde::Serialize;
```

And then you'll need the serde(crate) attribute.